BuildingBlocks/src/BuildingBlocks.Domain/MetricStatus.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Enmeshed.BuildingBlocks.Domain.StronglyTypedIds;
+
+namespace Enmeshed.BuildingBlocks.Domain;
+public class MetricStatus
+{
+    public MetricStatus(MetricKey metricKey)
+    {
+        MetricKey = metricKey;
+    }
+
+    public MetricStatus(MetricKey metricKey, DateTime isExhaustedUntil)
+    {
+        MetricKey = metricKey;
+        IsExhaustedUntil = isExhaustedUntil;
+    }
```

Comment:
I assume that you only added these constructors so that you can test your changes as long as the other tasks are not done yet?

Reply:
Correct. I'll be waiting for new changes to let go of this, i.e., load these from the database directly.